### PR TITLE
[Overflow-35] Create Question Form

### PR DESCRIPTION
**Pull request by IanUrriza10**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-35
## Commands
```
cd frontend
npm i 
npm run dev
```
## Pre-conditions
Install new packages

## Expected Output
A Question Form with inputs for Title, Description and Tags
Form requirements are in @/QuestionForm/schema.tsx
Not satisfying the requirements when clicking the submit button generates an alert box 
before the submit button

### CODE REVIEW

File: frontend/components/atoms/Editor/index.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      modules={modules}
+      formats={formats}
+      style={{ height: "10rem" }}
+      onChange={handleEditor}
+      id="descriptionInput"
+    />
+  );
+};
+
+export default Editor;
```
Comment (jescabillas) [inline frontend/components/atoms/Editor/index.tsx:76]: Redundant component, please use the existing editor component from other merged PR's

File: frontend/components/organisms/QuestionForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+              <span className="font-medium">
+                Ensure that these requirements are met:
+              </span>
+              <ul className="mt-1.5 ml-4 list-disc list-inside">
+                {Object.values(errors).map((errorText, index) => {
+                  return <li key={index}>{errorText.message}</li>;
+                })}
+              </ul>
+            </div>
+          </div>
```
Comment (jescabillas) [inline frontend/components/organisms/QuestionForm/index.tsx:107]: Please create components for this

File: frontend/components/organisms/QuestionForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from "react";
+import "react-quill/dist/quill.snow.css";
```
Comment (presyos) [inline frontend/components/organisms/QuestionForm/index.tsx:2]: Please remove unused import

File: frontend/components/organisms/QuestionForm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                  return <li key={index}>{errorText.message}</li>;
+                })}
+              </ul>
+            </div>
+          </div>
+        )}
+        <div className="Submit self-center w-4/5 p-6">
+          <button
+            type="submit"
+            className="border-2 border-red-400 rounded-lg text-red-400 p-2 float-right px-4 "
```
Comment (presyos) [inline frontend/components/organisms/QuestionForm/index.tsx:112]: Please use the existing button in atoms

**Review by presyos — CHANGES_REQUESTED** [review]

**Review by jescabillas — COMMENTED** [review]
Reviewed: 44mins
Comments: 2
Iteration: 1

**Comment by IanUrriza10** [comment]
3hrs refactor

**Review by jescabillas — APPROVED** [review]

**Review by presyos — APPROVED** [review]

**Pull request merged**